src/pages/home/report/ReportActionsView.js

```diff
@@ -23,6 +23,7 @@ import EmojiPicker from '../../../components/EmojiPicker/EmojiPicker';
 import * as ReportActionsUtils from '../../../libs/ReportActionsUtils';
 import * as ReportUtils from '../../../libs/ReportUtils';
 import reportPropTypes from '../../reportPropTypes';
+import getIsReportFullyVisibleUtil from '../../../libs/getIsReportFullyVisible';
```

Comment:
Can we remove `Util` here? Looks like in the ReportScreen we just use `import getIsReportFullyVisible from...`

```suggestion
import getIsReportFullyVisible from '../../../libs/getIsReportFullyVisible';
```